Prototope/DisplayLink.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  DisplayLink.swift
+//  Prototope
+//
+//  Created by Jason Brennan on 2015-08-10.
+//  Copyright (c) 2015 Khan Academy. All rights reserved.
+//
+
+import AppKit
+
+
+typealias HeartbeatDisplayLinkCallback = (sender: SystemDisplayLink) -> Void
+
+/** Crappy wrapper around CVDisplayLink to act pretty close to a CADisplayLink. Only OS X kids get this. */
+class DisplayLink: NSObject {
+
+	private let displayLink:CVDisplayLink? = {
+		var linkRef:Unmanaged<CVDisplayLink>?
```

Comment:
Yeah, this is a mess. For now I’d like to leave it as is because I’m going to need to re-write it for Swift 2. I’ve left a note that the class is currently kind of busted.